### PR TITLE
Add recipe for fiscalyear

### PR DESCRIPTION
**Pull request by CurtLH**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by CurtLH** [comment]
This recipe is ready for review @conda-forge/help-python.

File: recipes/fiscalyear/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (marcelotrevisani) [inline recipes/fiscalyear/LICENSE]: You license is already being packaged in Pypi, you don't need this file, please delete it

File: recipes/fiscalyear/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: 3fde4a12eeb72da446beb487e078adf1223a92d130520e589b82d7d1509701a2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/fiscalyear/meta.yaml]: Please, add ``setuptools`` here

File: recipes/fiscalyear/LICENSE (hunk 1 above)
Comment (CurtLH) [reply recipes/fiscalyear/LICENSE]: I do not believe the license is included in the tarball on Pypi.  After removing it from the recipe, the build failed with the following error:

```
ValueError: License file given in about/license_file (C:\projects\staged-recipes\recipes\fiscalyear\LICENSE) does not exist in source root dir or in recipe root dir (with meta.yaml)
```

Comment (marcelotrevisani) [reply recipes/fiscalyear/LICENSE]: That is true, sorry. I might downloaded the wrong project when I was checking it. My apologies about it.
Could you please open a pull request or create an issue on this github's project to also package the LICENSE?
Please revert your last commit then, sorry

Comment (CurtLH) [reply recipes/fiscalyear/LICENSE]: PR to add license to the tarball: https://github.com/adamjstewart/fiscalyear/pull/7

**Comment by adamjstewart** [comment]
Feel free to add me as a maintainer!

**Pull request merged**